src/plugins/saved_objects_management/public/management_section/objects_table/components/delete_confirm_modal.tsx

```diff
@@ -84,26 +88,42 @@ export const DeleteConfirmModal: FC<DeleteConfirmModalProps> = ({
         </EuiModalHeaderTitle>
       </EuiModalHeader>
       <EuiModalBody>
-        {undeletableObjects.length > 0 && (
+        {hiddenObjects.length + managedObjects.length > 0 && (
           <>
             <EuiCallOut
               data-test-subj="cannotDeleteObjectsConfirmWarning"
               title={
                 <FormattedMessage
                   id="savedObjectsManagement.objectsTable.deleteConfirmModal.cannotDeleteCallout.title"
-                  defaultMessage="Some objects cannot be deleted"
+                  defaultMessage="Some objects have been excluded"
                 />
               }
               iconType="warning"
               color="warning"
             >
-              <p>
-                <FormattedMessage
-                  id="savedObjectsManagement.objectsTable.deleteConfirmModal.cannotDeleteCallout.content"
-                  defaultMessage="{objectCount, plural, one {# object is} other {# objects are}} hidden and cannot be deleted. {objectCount, plural, one {It was} other {They were}} excluded from the table summary."
-                  values={{ objectCount: undeletableObjects.length }}
-                />
-              </p>
+              {hiddenObjects.length > 0 && (
+                <p
+                  css={css`
+                    margin-block-end: 0 !important;
+                  `}
+                >
```

Comment:
🙈 In theory we should be able to do this without inline css just with EUI components, don't we?

Reply:
If the concern is around inline CSS (?) this uses emotion so it generates a class for this element.